src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -389,5 +389,13 @@
     <bean id="weeklyEmailStep" class="org.kuali.kfs.sys.batch.WeeklyEmailStep" parent="step"
           p:actionListEmailService-ref="enActionListEmailService"
     />
+    
+    <!-- backport changes for redis -->
+    <bean id="menuService-parentBean" class="org.kuali.kfs.sys.service.MenuService" p:dao-ref="menuDao"
```

Comment:
Because this is a "parent" bean, it should be defined as an abstract bean.

Reply:
updated